### DISCUSSION TITLE
Port Change

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#3184] [Help with using Postal]
Im try to change port on postal but it still show in port 25 only  below this my config

smtp_server:
   default_bind_address: "::"
   port: 587
   tls_enabled: true
   tls_certificate_path: /opt/postal/config/smtp.cert
   tls_private_key_path: /opt/postal/config/smtp.key
 # tls_ciphers:
  ssl_version: SSLv23

**Comment by willpower232** [comment]
Did you restart Postal after changing the config?

Please also note that Postal only supports explicit SSL through STARTTLS so its likely that using port 587 won't work for you.

**Reply by arunkuttytry** [reply]
yes after change im stop and start the postal but still same, also please guid me how can i do this , i want configure salesforce

**Reply by willpower232** [reply]
Where are you seeing it as port 25? Can  you send a screenshot?
